alvr/client_openxr/src/lib.rs

```diff
@@ -636,9 +645,7 @@ pub fn entry_point() {
             swapchains[0].wait_image(xr::Duration::INFINITE).unwrap();
             swapchains[1].wait_image(xr::Duration::INFINITE).unwrap();
 
-            if !hardware_buffer.is_null() {
-                alvr_client_opengl::render_stream(hardware_buffer, [left_view_idx, right_view_idx]);
-            }
+            alvr_client_opengl::render_stream(hardware_buffer, [left_view_idx, right_view_idx]);
```

Comment:
not sure if we can have a case where `!hardware_buffer.is_null() && history_views.is_some()`
we probably want to force a re-projection in that case?
either way, I used it with this logic for ~1h yesterday and i did not see any flickers.

Reply:
I think this is good, no worries.